Makefile

```diff
@@ -30,7 +30,7 @@ redis-cli:
 	docker-compose run redis redis-cli -h redis
 
 run:
-	docker-compose run web airflow $(COMMAND)
+	docker-compose run -e AWS_SECRET_ACCESS_KEY -e AWS_ACCESS_KEY_ID web airflow $(COMMAND)
```

Comment:
Hm, I'm going to punt on this for now, since I'm not sure if we want to automatically pass the through more generally when spinning up the stack. I'll file an issue to follow up.

Reply:
Filed #178